app/assets/javascripts/editorToolbar.js

```diff
@@ -84,22 +57,18 @@ $(function() {
           //   1. when user drag-and-drops image
           //   2. when user clicks on upload button.
         start: function(e) {
+          $E.setState(e.target.dataset.formId); // string that is: "main", "reply-123", "edit-123" etc.
           $(e.target).removeClass('hover');
-          // for click-upload-button scenarios, it's important to set $D.selected here, because the 'drop' listener above doesn't run in those:
-          $D.selected = $(e.target).closest('div.comment-form-wrapper');
-          // the above line is redundant in drag & drop, because it's assigned in 'drop' listener too.
-          // on /wiki/new & /wiki/edit, $D.selected will = undefined from this assignment
-          elem = $($D.selected).closest('div.comment-form-wrapper').eq(0);
-          elem.find('.progress-bar-container').eq(0).show();
-          elem.find('.uploading-text').eq(0).show();
-          elem.find('.choose-one-prompt-text').eq(0).hide();
+          console.log($("#image-upload-progress-container-" + $E.commentFormID));
```

Comment:
Remove `console.log`

Reply:
Thank you, yeah, I didn't catch this one! I think I remove this in a later PR, but just in case, adding it to my list.